web/src/engine/dom-utils/src/stylesheets.ts

```diff
@@ -106,8 +108,9 @@ export class StylesheetManager {
     // Modern browsers: use WOFF, TTF and fallback finally to SVG. Don't provide EOT
```

Comment:
No SVG fonts! They're as dead as EOT.
```suggestion
    // Modern browsers: use WOFF2, WOFF, and fallback finally to TTF. SVG, EOT not supported.
```

See also lines 122-127. Perhaps should be done as a separate issue.